### DISCUSSION TITLE
EdgeID Retrieval

### DISCUSSION THREAD

**Discussion by ShrenikaSrinivas** [eclipse/mosaic#101] [Q&A] [answered]
Hello, I need to make use of Edge Id for the application, could you please tell me where I could find the API which returns me Edge ID from MOSAIC

**Comment by kschrab** [comment, marked as answer]
You can use `getOs().getRoadPosition().getConnectionId();` for that. The term "Connection" in MOSAIC is equivalent to the term "Edge" in SUMO.
